CONTRIBUTING.md

```diff
@@ -127,7 +127,36 @@ review
 
 Before sending a pull request, it should include tests if there are logic
 changes, copyright headers in every file, and a commit message following the
-conventions below.
+conventions in "Commit messages" section below.
+
+A pull request can be opened from a branch within the repository or from a
+fork. External contributors are only able to open pull requests from forks,
+but team members with write access can choose to open a pull request from a
+repository branch.
+
+### Pull request from a fork
+
+If you open a pull request from a personal fork, you should allow repository
+maintainers to make edits to your fork by turning on
+"Allow edits from maintainers".
+
+### Pull request from a branch
+
```

Comment:
Add link to https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#changing-the-branch-range-and-destination-repository for details